libs/alerts/src/main/java/org/codice/ddf/system/alerts/SystemNotice.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.system.alerts;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * SystemNotice is the property map class used by DDF collectors when sending system notifications to
+ * the Decanter topic 'decanter/collect/*'. This class defines all the common keys used in system
+ * notification events and provides convenient helpers for accessing them.
+ */
+public class SystemNotice extends HashMap<String, Object> {
+    //topics
+    public static final String SYSTEM_NOTICE_BASE_TOPIC = "decanter/collect/";
+
+    public static final String SYSTEM_NOTICE_EVENT_TYPE = "system-notice";
+
+    //common decanter fields
+    public static final String EVENT_TYPE_KEY = "type";
```

Comment:
❗️ If we keep these constants, we need to clearly document what they mean, what they're used for, if they have any constraints, which ones are used as part of the unique identifier, etc.